composer.json

```diff
@@ -37,17 +37,17 @@
     ],
 
     "require": {
-        "php": ">=5.5",
-        "league/climate": "^2.0|^3.0",
+        "php": "^5.6 || ^7.0",
+        "league/climate": "^2.0 || ^3.0",
         "symfony/console": "^2.0",
         "symfony/process": "^2.1"
     },
 
     "require-dev": {
         "mockery/mockery": "^0.9",
-        "phpunit/phpunit": "^4.0",
+        "phpunit/phpunit": "^5.1",
         "sensiolabs/security-checker": "^3.0",
-        "squizlabs/php_codesniffer": "^2.0"
+        "squizlabs/php_codesniffer": "^2.5"
```

Comment:
Well "up to date" implies that something would be required in this new version. Using `^2.0` would install 2.5 (or whatever is newest 2.x) automatically. Unless there's a new major version, there is no need to change versions.